reflex/ivars/object.py

```diff
@@ -488,89 +468,144 @@ def _get_all_var_data(self) -> ImmutableVarData | None:
+    #     )
+
     @cached_property
     def _cached_var_name(self) -> str:
         """The name of the operation.
 
         Returns:
             The name of the operation.
         """
-        return f"Object.keys({self.value._var_name})"
-
-
-class ObjectValuesOperation(ObjectToArrayOperation):
-    """Operation to get the values of an object."""
+        return f"Object.keys({str(self._value)})"
 
-    def __init__(
-        self,
+    @classmethod
+    def create(
+        cls,
         value: ObjectVar,
         _var_data: VarData | None = None,
-    ):
-        """Initialize the object values operation.
+    ) -> ObjectKeysOperation:
+        """Create the object keys operation.
 
         Args:
             value: The value of the operation.
             _var_data: Additional hooks and imports associated with the operation.
+
+        Returns:
+            The object keys operation.
         """
-        super(ObjectValuesOperation, self).__init__(
-            value, List[value._value_type()], _var_data
+        return cls(
+            _var_name="",
+            _var_type=List[str],
+            _var_data=ImmutableVarData.merge(_var_data),
+            _value=value,
         )
 
+
+class ObjectValuesOperation(ObjectToArrayOperation):
+    """Operation to get the values of an object."""
+
     @cached_property
     def _cached_var_name(self) -> str:
         """The name of the operation.
 
         Returns:
             The name of the operation.
         """
-        return f"Object.values({self.value._var_name})"
+        return f"Object.values({self._value._var_name})"
 
-
-class ObjectEntriesOperation(ObjectToArrayOperation):
-    """Operation to get the entries of an object."""
-
-    def __init__(
-        self,
+    @classmethod
+    def create(
+        cls,
         value: ObjectVar,
         _var_data: VarData | None = None,
-    ):
-        """Initialize the object entries operation.
+    ) -> ObjectValuesOperation:
+        """Create the object values operation.
 
         Args:
             value: The value of the operation.
             _var_data: Additional hooks and imports associated with the operation.
+
+        Returns:
+            The object values operation.
         """
-        super(ObjectEntriesOperation, self).__init__(
-            value, List[Tuple[value._key_type(), value._value_type()]], _var_data
+        return cls(
+            _var_name="",
+            _var_type=List[value._value_type()],
+            _var_data=ImmutableVarData.merge(_var_data),
+            _value=value,
         )
 
+
+class ObjectEntriesOperation(ObjectToArrayOperation):
+    """Operation to get the entries of an object."""
+
     @cached_property
     def _cached_var_name(self) -> str:
         """The name of the operation.
 
         Returns:
             The name of the operation.
         """
-        return f"Object.entries({self.value._var_name})"
+        return f"Object.entries({self._value._var_name})"
```

Comment:
same here, prefer `str(self._value)` unless i'm missing something about why this needs to be different here